api/v1alpha1/etcdcluster_types.go

```diff
@@ -67,6 +67,9 @@ type EtcdClusterStatus struct {
 	// Members contains information about each member from the etcd cluster.
 	// +optional
 	Members []EtcdMember `json:"members"`
+
+	// APIVersion contains the cluster API version
+	APIVersion string `json:"apiVersion"`
```

Comment:
This looks a lot like a Kubernetes `apiVersion` - could we name this differently?

Reply:
I'll call it ClusterVersion, which is what etcdclientv3 calls it.